client/lib/plans/index.js

```diff
@@ -204,3 +215,33 @@ export function plansLink( url, site, intervalType ) {
 
 	return url;
 }
+
+export function applyTestFiltersToPlansList( planName ) {
+	const filteredPlanConstantObj = Object.assign( {}, plansList[ planName ] );
+	let filteredPlanFeaturesConstantObj = getPlanFeaturesConstantObj( plansList[ planName ].getFeatures() );
+
+	if ( ! isWordadsInstantActivationEnabled() ) {
+		filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
+			( _, key ) => key !== WORDADS_INSTANT
```

Comment:
`_` is actually the object value. Let's avoid using `_` since reading this at a glance I would assume you're importing the full lodash object here.